packages/gasket-plugin-middleware/lib/utils.js

```diff
@@ -69,7 +70,14 @@ function applyMiddlewaresToApp(app, middlewares, middlewarePattern) {
     return;
   }
 
-  middlewares.forEach(({ handler, paths }) => {
+  middlewares.forEach((layer) => {
+    // Normalize: allow either direct middleware function or object with { handler, paths }
+    const paths = layer.paths;
+    const handler = typeof layer === 'function' ? layer : layer.handler;
```

Comment:
Can't `layer` also be an Array of middleware functions?

Reply:
Yes it could, good catch.

Reply:
Great catch! Thanks @jpage-godaddy !